packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/ReferencesButton.tsx

```diff
@@ -28,28 +28,30 @@ type TReferencesButtonProps = {
   focusSpan: (spanID: string) => void;
 };
 
+// ReferencesButton is displayed as a menu at the span level.
+// Example: https://github.com/jaegertracing/jaeger-ui/assets/94157520/2b29921a-2225-4a01-9018-1a1952f186ef
```

Comment:
![Example](https://github.com/jaegertracing/jaeger-ui/assets/94157520/2b29921a-2225-4a01-9018-1a1952f186ef)

(Used some dummy data to visualize this component)